lib/normalize-option.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var valueOrFunction = require('value-or-function');
+
+module.exports = function normalizeOption(types, defaultValue, value) {
```

Comment:
Or maybe `value-or-function` should support a function along with strings as the first argument; if we did that, we could properly support `Date or timestamp` as the type for `since`.  Thoughts?